spikeextractors/extractors/tridescloussortingextractor/tridescloussortingextractor.py

```diff
@@ -47,4 +47,4 @@ def get_unit_spike_train(self, unit_id, start_frame=None, end_frame=None):
             spike_times = spike_times[spike_times >= start_frame]
         if end_frame is not None:
             spike_times = spike_times[spike_times < end_frame]
-        return spike_times.copy()  #�copy avoid reference to the unerlying memmap
+        return spike_times.copy()  #�copy avoid reference to the unerlying memmap
```

Comment:
?

Reply:
I'll fix this in the next pull request I make. Weird!